frontend/package.json

```diff
@@ -123,7 +123,7 @@
   "scripts": {
     "preinstall": "npx npm-force-resolutions",
     "start": "react-scripts start",
-    "build": "cross-env CI=true react-scripts build",
+    "build": "cross-env CI=true INLINE_RUNTIME_CHUNK=false react-scripts build",
```

Comment:
by default CRA apps inject a script tag into the index.html at production build time. This flag disables this behaviour, which allows us to use a content security policy that does not allow inline scripts.